src/jaguar.toit

```diff
@@ -171,6 +171,9 @@ install_program program_size/int reader/reader.Reader defines/Map -> none:
     if not timeout: timeout = Duration --s=10
     disabled = true
 
+  run_boot := false
+  defines.get "run.boot" --if_present=: if it: run_boot = true
```

Comment:
```suggestion
  run_boot := defines.get "run.boot" --if_absent=: false
```

Reply:
Yeah, I thought it would be better to handle the case where `it` was null, but it makes no sense here. Thanks!